common_types/src/datum.rs

```diff
@@ -347,6 +382,14 @@ pub enum Datum {
     Int16(i16),
     Int8(i8),
     Boolean(bool),
+    /// The Date representing the elapsed time since UNIX epoch in days(32 bits)
+    /// It is mapped to [`arrow::datatypes::DataType::Date32`].
```

Comment:
```suggestion
    /// Date represents the elapsed days since UNIX epoch.
    ///
    /// It is mapped to [`arrow::datatypes::DataType::Date32`].
```